tests/NoExceptions/NoExceptions.cpp

```diff
@@ -48,6 +54,7 @@ int main(int /*argc*/, char ** /*argv*/)
   float queuePriority = 0.0f;
   vk::DeviceQueueCreateInfo deviceQueueCreateInfo(vk::DeviceQueueCreateFlags(), static_cast<uint32_t>(graphicsQueueFamilyIndex), 1, &queuePriority);
   vk::UniqueDevice device = physicalDevices[0].createDeviceUnique(vk::DeviceCreateInfo(vk::DeviceCreateFlags(), 1, &deviceQueueCreateInfo)).value;
+  VULKAN_HPP_DEFAULT_DISPATCHER.init(*device);
```

Comment:
Really not needed, because below "device" functions already resolved during `VULKAN_HPP_DEFAULT_DISPATCHER.init(*instance);`.

Reply:
You're right, that's not really needed. But it doesnt hurt either.